ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/server/KsqlRestConfig.java

```diff
@@ -126,10 +126,13 @@ public class KsqlRestConfig extends AbstractConfig {
 
   public static final String SSL_STORE_TYPE_JKS = "JKS";
   public static final String SSL_STORE_TYPE_PKCS12 = "PKCS12";
+  public static final String SSL_STORE_TYPE_BCFKS = "BCFKS";
+
   public static final ConfigDef.ValidString SSL_STORE_TYPE_VALIDATOR =
       ConfigDef.ValidString.in(
           SSL_STORE_TYPE_JKS,
-          SSL_STORE_TYPE_PKCS12
+          SSL_STORE_TYPE_PKCS12,
+          SSL_STORE_TYPE_BCFKS
```

Comment:
Isn't there a unit test you need to adapt?

Reply:
> Isn't there a unit test you need to adapt?

For the other two we don't have any.

Reply:
OK

I think you need to adapt `SSL_KEYSTORE_TYPE_DOC` and `SSL_TRUSTSTORE_TYPE_DOC`, though.